tests/test_gitautosync.py

```diff
@@ -3,10 +3,10 @@
 import subprocess
 import random
 import string
-from gitautosync.gitautosync import GitAutoSync
+from gitautosync import GitAutoSync
 
 
-class TestGitAutoSync(object):
+class TestGitAutoSync:
```

Comment:
I think that's a great idea! Especially if we can test the handlers for concurrent use (since we want them to only do one sync request at a time).

Can we merge this first, and then can you add more tests? :D